src/OSPSuite.Core/Domain/Mappers/EventAssignmentBuilderToEventAssignmentMapper.cs

```diff
@@ -52,7 +58,7 @@ private EventAssignment createAssignment(EventAssignmentBuilder assignmentBuilde
          assignment.ObjectPath = assignmentBuilder.ObjectPath.Clone<ObjectPath>();
          assignment.UseAsValue = assignmentBuilder.UseAsValue;
 
-         simulationBuilder.AddBuilderReference(assignment, assignmentBuilder);
```

Comment:
I have removed this Builder reference and only use the _entitryTracker stuff instead